### ISSUE TITLE
KellyErrorEstimator should get the Mapping

### ISSUE THREAD

**Issue by tjhei** [geodynamics/aspect#292]
See:
```
$ grep -r Kelly source/ include/
source/mesh_refinement/velocity.cc:      KellyErrorEstimator<dim>::estimate (this->get_dof_handler(),
source/mesh_refinement/temperature.cc:      KellyErrorEstimator<dim>::estimate (this->get_dof_handler(),
source/mesh_refinement/composition.cc:          KellyErrorEstimator<dim>::estimate (this->get_dof_handler(),
```

add mapping as the first argument and check how tests change.

**Comment by gassmoeller** [comment]
This is closed, right?

**Comment by tjhei** [comment]
closed, see #317
